### ISSUE TITLE
Add to StaticGen

### ISSUE THREAD

**Issue by honzajavorek** [pyvec/elsa#56]
https://www.staticgen.com/ has a list of static site generators. It lists also Flask-Frozen. Why not to add Elsa?

**Comment by hroncok** [comment]
Could you please do it?

**Comment by honzajavorek** [comment]
Yes! I just wanted to check first whether you think it's a good idea :)

**Comment by hroncok** [comment]
https://www.staticgen.com/elsa
